docs/src/test-parallel-js.md

```diff
@@ -96,6 +96,10 @@ export default defineConfig({
 You can annotate inter-dependent tests as serial. If one of the serial tests
 fails, all subsequent tests are skipped. All tests in a group are retried together.
 
+:::note
```

Comment:
Serial more is discouraged and hence is a fraction of the testing use cases. Extension testing in serial mode is even smaller fraction. Extension testing that uses fixtures in serial mode is even smaller population. I don't think we should bother serial mode users with this information.

Reply:
We would be open to a PR that improves the error message though!